### ISSUE TITLE
Hosting on GH pages

### ISSUE THREAD

**Issue by averagegatsby** [maxvoltar/photo-stream#77]
Hi y'all, 

I went with the "slightly-less-easy-but-still-totally-doable way". Then, ended up going back, forking and clearing the directory as instructed.

Everything appears fine when hosting locally, but when setting up on GitHub Pages (site), I get this weird error:
<img width="788" alt="Screen Shot 2020-06-12 at 5 32 20 PM" src="https://user-images.githubusercontent.com/66851307/84555638-ad5ec400-acd2-11ea-8fcb-95a6f74051f4.png">

I was hosting another full jekyll project on the GitHub "account site" which was having issues with relative paths. I've been wondering if this was a similar issue for the past 4 hours. I also tried removing feed.xml and the Dockerfile to see if the error resolved itself (It still works fine on local host, but shows without images in GitHub Pages).

Any help would be much appreciated. Thank you 🙏

**Comment by victortokaito** [comment]
Is anyone able to host this on GitHub Pages?

**Comment by Julian** [comment]
I just found this project (which looks great!) and yeah managed to get it set up with GH Pages fairly easily.

Here's the GH Action I'm using to publish: https://github.com/Julian/photography/blob/master/.github/workflows/publish.yml

**Comment by Julian** [comment]
(I completely ignored the Dockerfile FWIW and just installed Ruby, bundler, and went to town.)

**Comment by TianZonglin** [comment]
> I just found this project (which looks great!) and yeah managed to get it set up with GH Pages fairly easily.
> 
> Here's the GH Action I'm using to publish: https://github.com/Julian/photography/blob/master/.github/workflows/publish.yml

Hello Julian, I also want to use the gh-pages to host this photo album, and I just forked the project then copied your action-scripts, I think it could be worked, but the results/content of the 'gh-pages branch' is absolutely different compared with yours, I'm really wondering how this could be made, did you make some additional configurations to achieve this? 

the action-scripts: https://github.com/TianZonglin/images/blob/master/.github/workflows/jekyll.yml
the content of pages branch: https://github.com/TianZonglin/images/tree/gh-pages

thanks for any possible help :)

**Comment by Julian** [comment]
You've got my CNAME still in there: https://github.com/TianZonglin/images/blob/master/.github/workflows/jekyll.yml#L47, so probably at least that'd need updating.

(But no what's there is exactly what powers my site at this point, so it's running for me: https://photos.grayvines.com/)

**Comment by TianZonglin** [comment]
yes I see it, but it shouldn't affect the result of github-actions, the point is I can't get the gh-pages-branch like yours, it quiet strange..
